### PR TITLE
fix: ensure directories we create as root are chown-ered to user

### PR DESCRIPTION
**Pull request by xavierchanth**
**- What I did**

Still encountering directory ownership issues with the installer, so I implemented a cleanup step which will call chown if the directory didn't exist prior to installation.

Also through a step in to print_env which will tell us the ssh status, useful for future debugging.

**- How I did it**

**- How to verify it**

**- Description for the changelog**
fix: ensure directories we create as root are chown-ered to user

### CODE REVIEW

File: packages/dart/sshnoports/bundles/universal.sh [hunk 1]
```diff
@@ -804,6 +843,22 @@ done
   fi
 }
 
+chown_dir() {
+  echo "$1 was created by this installer, ensuring that it is owned by $user"
+  chown -R $user:$user "$1" || chown -R $user "$1"
+}
+
+cleanup() {
+  if as_root; then
```
Comment (xavierchanth) [inline packages/dart/sshnoports/bundles/universal.sh]: Only run this step if we were running as root,

**Review by cpswan — DISMISSED** [review]

**Comment by xavierchanth** [comment]
Doing a sanity test before I merge

**Comment by xavierchanth** [comment]
Okay so here are the cases mapped to behaviour during `cleanup`:

- dir didn't exist, but was created   : calls `chown_dir` which gives ownership to the user.
- dir didn't exist, and still doesn't : calls `chown_dir` which silently does nothing.
- dir already existed                 : doesn't call `chown_dir` which obviously does nothing.

**Review by Xlin123 — APPROVED** [review]
lgtm

**Pull request merged**